tests/libtest/thrift_client_to_db.py

```diff
@@ -95,16 +95,18 @@ def __exit__(self, type, value, tb):
 
 class CCViewerHelper(ThriftAPIHelper):
 
-    def __init__(self, host, port, product, endpoint,
+    def __init__(self, protocol, host, port, product, endpoint,
                  auto_handle_connection=True, session_token=None):
         # Import only if necessary; some tests may not add this to PYTHONPATH.
         from libcodechecker import session_manager
         from codeCheckerDBAccess_v6 import codeCheckerDBAccess
         from codeCheckerDBAccess_v6.constants import MAX_QUERY_SIZE
 
         self.max_query_size = MAX_QUERY_SIZE
-        transport = THttpClient.THttpClient(
-            host, port, '/' + product + '/v' + VERSION + endpoint)
+        url = protocol+"://"+host+":"+str(port) +\
+            '/' + product + '/v' + VERSION + endpoint
+        print ("setup viewer client:"+url)
```

Comment:
Is this print intended? If so, maybe having a space after colon and starting with capital letter would be great.